services/updateUser.service.js

```diff
@@ -0,0 +1,15 @@
+const User = require("../models/user.model");
+
+exports.updateUserService = async (body) => {
+  try {
+    await User.updateOne(
+      { email: body.email },
+      {
+        $set: body,
+      }
+    );
+    return res.status(200).json("updatedUser");
+  } catch (err) {
+    return err;
+  }
```

Comment:
try catch should be included in controller, not in service

Reply:
done